.travis.yml

```diff
@@ -46,6 +46,13 @@ jobs:
     - name: storybook-smoke-test (node@production)
       <<: *node-prod
       script: yarn storybook-smoke-test
+    - name: yarn serve (node@production)
+      <<: *node-prod
+      script:
+        # `yarn build` takes some time so let's simulate it instead.
+        - mkdir build && touch build/index.html
+        - yarn serve &
+        - sleep 120 && pkill -f node
```

Comment:
Maybe only sleep for 30 seconds? If we catch a failure in the wild where TS hadn't finished yet, we could increase the sleep time.

Reply:
👍